EIPS/eip-7923.md

```diff
@@ -154,6 +157,10 @@ index bf34fbee..f63b7460 100644
 +
 +        for page in range(start_page, end_page + 1):
 +            if page not in self._memory.pages:
++                self.transaction_context.num_pages += 1
++                if self.transaction_context.num_pages > TRANSACTION_MAX_PAGES:
++                    raise VMError("Out Of Memory")
```

Comment:
Before this error, the page should still be allocated to `self._memory.pages` otherwise the assertion below will fail (also commented there)

Reply:
Or maybe easier: first do the check `self.transaction_context.num_pages >= TRANSACTION_MAX_PAGES:` (should be equal, we are at max) and if this passes then update `self.transaction_context.num_pages++`

Reply:
very nice, thanks
https://github.com/ethereum/EIPs/commit/d951ba52246f5c43ebc402ebf8a4b7538ecafac2